packages/orchestration/src/exos/local-orchestration-account.js

```diff
@@ -47,10 +50,12 @@ const { Fail } = assert;
 const HolderI = M.interface('holder', {
   ...orchestrationAccountMethods,
   getPublicTopics: M.call().returns(TopicsRecordShape),
-  delegate: M.call(M.string(), AmountShape).returns(M.promise()),
-  undelegate: M.call(M.string(), AmountShape).returns(M.promise()),
-  withdraw: M.callWhen(AmountShape).returns(PaymentShape),
-  executeTx: M.callWhen(M.arrayOf(M.record())).returns(M.arrayOf(M.record())),
+  delegate: M.call(M.string(), AmountShape).returns(VowShape),
+  undelegate: M.call(M.string(), AmountShape).returns(VowShape),
+  withdraw: M.call(AmountShape).returns(NetworkShape.Vow$(PaymentShape)),
```

Comment:
`Network` is a distraction here.

I suggest doing something like `const { Vow$ } = NetworkShape;` above.